src/views/admin/rollen/RolleDetailsView.vue

```diff
@@ -92,67 +99,31 @@
 
   const translatedSystemrechte: ComputedRef<TranslatedObject[]> = computed(() => {
     const systemrechte: Array<RollenSystemRecht> = Array.from(rolleStore.currentRolle?.systemrechte || []);
-    if (!systemrechte.length) {
-      return [{ value: '---', title: '---' }];
-    }
     return systemrechte.map((systemrechtKey: RollenSystemRecht) => ({
       value: systemrechtKey,
       title: t(`admin.rolle.mappingFrontBackEnd.systemrechte.${systemrechtKey}`),
     }));
   });
 
-  const validationSchema: TypedSchema = getValidationSchema(t);
-
-  const vuetifyConfig = (state: {
-    errors: Array<string>;
-  }): { props: { error: boolean; 'error-messages': Array<string> } } => getVuetifyConfig(state);
-
-  // eslint-disable-next-line @typescript-eslint/typedef
-  const { defineField, handleSubmit, isFieldDirty, resetForm, setFieldValue } = useForm({
-    validationSchema,
-  });
+  const formContext: FormContext<RolleFormType, RolleFormType> = useForm<RolleFormType>({ validationSchema });
+
+  const {
+    selectedAdministrationsebene,
+    selectedAdministrationsebeneProps,
+    selectedRollenArt,
+    selectedRollenArtProps,
+    selectedRollenName,
+    selectedRollenNameProps,
+    selectedMerkmale,
+    selectedMerkmaleProps,
+    selectedServiceProviders,
+    selectedServiceProvidersProps,
+    selectedSystemRechte,
+    selectedSystemRechteProps,
+  }: RolleFieldDefinitions = getRolleFieldDefinitions(formContext);
+
+  const isFormDirty: boolean = !isEditActive.value ? getDirtyState() : false;
```

Comment:
Wird das je aktualisiert?

Reply:
Habe es jetzt gefixed. Dirtiness is reaktiv.